components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/constant/OrganizationManagementConstants.java

```diff
@@ -751,7 +751,10 @@ public enum ErrorMessages {
                 "Server encountered an error while updating organization user associations for the user."),
         ERROR_CODE_ERROR_CHECK_ORGANIZATION_USER_ASSOCIATIONS("65144",
                 "Unable to check if organization user associations exist.",
-                "Server encountered an error while checking organization user associations for the user.");
+                "Server encountered an error while checking organization user associations for the user."),
+        ERROR_CODE_ERROR_RETRIEVING_ORGANIZATION_NAMES_IN_BATCH("60100",
```

Comment:
Check whether this error code is duplicated or not

Reply:
Also, since this is an error return for server exception, error code should be 65xxx
Use the next available one